webapp/tests/karma/ts/services/geolocation.service.spec.ts

```diff
@@ -137,13 +124,14 @@ describe('Geolocation service', () => {
         heading: 6,
         speed: 7,
       };
-      $window.navigator.geolocation.watchPosition.callsFake((success, failure) => {
+      // @ts-ignore
+      window.navigator.geolocation.watchPosition.callsFake((success, failure) => {
         success({coords: first});
         failure({code: 2, message: 'some later error'});
       });
 
       return service.init()().then(returned => {
```

Comment:
For this one and the following unit test:
I think it will look better if these functions are in separate line.
Also I tend to like better async / await to for unit test but these still work fine, something optional

Reply:
I just copied and pasted these tests and just fixed what was broken. Would you mind if I just leave them as they were?

Reply:
That’s okay, it's the original :)